### PR TITLE
partial refactor

### PR DESCRIPTION
**Pull request by jerryxfu**

### CODE REVIEW

File: src/AutoColor.java [hunk 1]
```diff
@@ -112,27 +113,27 @@ public static String colorize(String str, String regexDelim, String... hexColors
...
+        int hexIndex;
 
         while (index != strs.length) {
             // stop hexIndex from being too big
             hexIndex = index;
             if (hexIndex >= hexColors.length)
                 hexIndex = hexColors.length - 1;
             // create colored text
-            finalStr += obj.new Color(hexColors[hexIndex]).ANSIfy(setupModifiers(), strs[index], false);
+            finalStr += new Color(hexColors[hexIndex]).ANSIfy(setupModifiers(), strs[index], false);
```
Comment (Arnaud-5G) [inline src/AutoColor.java]: revert this

**Review by Arnaud-5G — CHANGES_REQUESTED** [review]

**Comment by jerryxfu** [comment]
image
See this? when i put it back it uh breaks for me :/

**Comment by jerryxfu** [comment]
uh ok how about you take care of merging this without errors thanks

**Comment by Arnaud-5G** [comment]
It's because your Color class is static

**Pull request closed without merging**